packages/esm-home-app/src/homepage-header/homepage-header.scss

```diff
@@ -0,0 +1,85 @@
+@use '@carbon/colors';
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.header {
+    @include type.type-style('body-compact-02');
```

Comment:
The indentation looks off for this entire file. It should default to 2 tab widths.